k8s/namespaces/xui/xui-mo-webapp/perftest.yaml

```diff
@@ -12,7 +12,8 @@ spec:
         FEATURE_TERMS_AND_CONDITIONS_ENABLED: false
         DEBUG: xuiNode:*,-xuiNode:auth:s2s
         SERVICES_MCA_PROXY_API: http://aac-manage-case-assignment-perftest.service.core-compute-perftest.internal
-        CASE_TYPES: FinancialRemedyConsentedRespondent
+        CASE_TYPES: FinancialRemedyConsentedRespondent,UNSPECIFIED_CLAIMS,MoneyClaimCase,DIVORCE,FinancialRemedyContested,FinancialRemedyMVP2,Asylum,Caveat,GrantOfRepresentation,StandingSearch,WillLodgement,CARE_SUPERVISION_EPO,Benefit
```

Comment:
The FinancialRemedyConsentedRespondent is a test case type, it needs to be removed as it will cause issues.